operator/src/main/java/org/keycloak/operator/crds/v2alpha1/StatusCondition.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.operator.crds.v2alpha1;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * @author Vaclav Muzikar <vmuzikar@redhat.com>
```

Comment:
don't you want to add yourself as the author, since it's a newly added file?

Reply:
I don't think I've ever used author tags before.  If that's a convention for the project, then I'll start adding them.

Reply:
That's not strictly necessary, as far as I'm aware, we don't have any conventions in this regard.